crates/bevy_reflect/src/func/return_type.rs

```diff
@@ -6,9 +6,9 @@ use crate::PartialReflect;
 /// [`DynamicFunctionMut`]: crate::func::DynamicFunctionMut
 #[derive(Debug)]
 pub enum Return<'a> {
-    /// The function returns nothing (i.e. it returns `()`).
-    Unit,
     /// The function returns an owned value.
+    ///
+    /// Owned values can be unit (`()`).
```

Comment:
Maybe something like:
```suggestion
    /// This includes functions that return nothing (i.e. they return `()`).
```